geoip2/webservice.py

```diff
@@ -83,7 +83,7 @@ class Client(object):
     """
 
     def __init__(self,
-                 user_id,
+                 account_id,
```

Comment:
Is there any concern that this might break backwards compatibility?

I am not very familiar with Python, but it looks like named parameters are supported:

```python
>>> client = geoip2.webservice.Client(account_id=1, license_key='test123')
>>> client = geoip2.webservice.Client(user_id=1, license_key='test123')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() got an unexpected keyword argument 'user_id'
```

Maybe it's uncommon enough that we don't need to worry though.

Reply:
Yeah, that is a good point. Calling it like that would be pretty uncommon for parameters that are not optional, but I'll see if I can come up with a solution that works.